pmpro-roles.php

```diff
@@ -366,13 +366,24 @@ public static function level_settings() {
 					<tr>
 						<th scope="row" valign="top"><label><?php esc_html_e( 'Roles', 'pmpro-roles' ); ?>:</label></th>
 						<td>
-							<div class="checkbox_box<?php if ( count( $editable_roles ) > 5 ) { ?> pmpro_scrollable<?php } ?>">
-              <input type="hidden" name="pmpro_roles_level_present" value="1" />
-								  <?php
+							<?php
+								// Build the selectors for the checkbox list based on number of levels.
+								$classes = array();
+								$classes[] = "pmpro_checkbox_box";
+								if ( count( $editable_roles ) > 5 ) {
+									$classes[] = "pmpro_scrollable";
+								}
+								$class = implode( ' ', array_unique( $classes ) );
+							?>
+							<div class="<?php echo esc_attr( $class ); ?>">
+								<input type="hidden" name="pmpro_roles_level_present" value="1" />
+								<?php
 									//New level, choose if they want to create a role for this level
 									if ( $_REQUEST['edit'] < 0 ) { ?>
-										<div class="clickable">
-											<input type='checkbox' name='pmpro_roles_level[pmpro_draft_role]' value='pmpro_draft_role' id='pmpro_draft_role' /> <label for='pmpro_draft_role'><?php _e('Create a new custom role for this membership level', 'pmpro-roles'); ?>
+										<div class="pmpro_clickable" style="border-bottom-width: 4px;">
```

Comment:
Should we add another class and put this border CSS into https://github.com/strangerstudios/pmpro-roles/blob/dev/admin.css?